modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Configuring mount namespace encapsulation
+
+You can configure mount namespace encapsulation so that a cluster runs with less resource overhead.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a file called `mount_namespace_config.yaml` with the following YAML to control whether encapsulation is enabled or disabled:
++
+[source,yaml]
+----
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: master
+    name: 99-kubens-master
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <1>
+          name: kubens.service
+  ---
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker
+    name: 99-kubens-worker
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <2>
+          name: kubens.service
+----
+<1> If `enabled` is set to true, encapsulation is in effect for all control plane (previously known as "master") nodes.
+<2> If `enabled` is set to false, encapsulation is disabled for all worker nodes.
```

Comment:
```suggestion
<2> If `enabled` is set to true, encapsulation is in effect for all worker nodes.
```

Reply:
We should add an additional note here that while this feature is in Technology Preview, the feature is disabled by default, and must be manually enabled.

Reply:
Done.